crypto/constant_time_test.c

```diff
@@ -57,14 +58,14 @@ static int test_binary_op_w(crypto_word (*op)(crypto_word a, crypto_word b),
                             int is_true) {
   crypto_word c = op(a, b);
   if (is_true && c != CONSTTIME_TRUE_W) {
-    fprintf(stderr,
+    printerr(
```

Comment:
These tests will pretty much never fail so I think we should just remove all the print statements completely, instead of jumping through hoops.